framework/src/auxkernels/MortarNodalAuxKernel.C

```diff
@@ -104,6 +109,12 @@ MortarNodalAuxKernelTempl<ComputeValueType>::compute()
   // We have to reinit the node for this variable in order to get the dof index set for the node
   _var.reinitNode();
 
+  // If the node doesn't have corresponding mortar segments, force the value assigned in this step
+  // to be zero. This can be useful when nodes initially do not project but will project at a
+  // different stage of the simulation
+  if (total_volume == 0)
+    total_volume = std::numeric_limits<Real>::max();
```

Comment:
why not rather do 

```
  if (total_volume == 0)
    value = 0;
  else
    value /= total_volume;
```
dividing by max real later is a bit of a detour IMO

Reply:
maybe a fuzzy equal?

Reply:
We are setting value/total_volume below